src/platform_impl/x11/mod.rs

```diff
@@ -26,10 +26,10 @@ pub struct GlobalHotKeyManager {
 }
 
 impl GlobalHotKeyManager {
-    pub fn new() -> crate::Result<Self> {
+    pub fn new() -> Self {
         let (thread_tx, thread_rx) = unbounded();
         std::thread::spawn(|| events_processor(thread_rx));
-        Ok(Self { thread_tx })
+        Self { thread_tx }
     }
```

Comment:
The `Result` here appears unnecessary and used.

This shouldn't cause any external changes. The goal is to make the API a bit cleaner.